sdk/src/utils.ts

```diff
@@ -243,20 +243,20 @@ export const convertArr = (arr: any[]): any[] => {
   })
 }
 
-export const assertError = async <T>(fn: Promise<T>, invariantError: InvariantError) => {
-  let exceptionThrown = false
-
+export const assertThrowsAsync = async (fn: Promise<any>, word?: InvariantError | string) => {
```

Comment:
you have to be consistent. one of two possible:
- `InvariantError | Tx | string`
- `string`